src/react-apps/applications/shared/__tests__/profileMenu.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import ProfileMenuComponent from '../src/navigation/main-header/profileMenu';
+
+describe('ProfileMenu', () => {
+  let profileMenu: any;
+  let rendered: any;
+
+  beforeEach(() => {
+    profileMenu = null;
+    rendered = null;
+  });
+
+  it('snapshot should match snapshot with default values', () => {
```

Comment:
Just `it('should match snapshot'), () => ...)`

Reply:
done